core-bundle/src/Resources/contao/elements/ContentModule.php

```diff
@@ -72,7 +78,14 @@ public function generate()
 			$responseTagger->addTags(array('contao.db.tl_content.' . $this->id));
 		}
 
-		return $objModule->generate();
+		$strBuffer = $objModule->generate();
+
+		if ($objStopwatch)
```

Comment:
<img width="772" alt="" src="https://user-images.githubusercontent.com/1192057/117451760-3910bd00-af43-11eb-8128-f51d5e900dad.png">

Reply:
I think you just have to add `$objStopwatch = null;` above so the variable is always defined.

Reply:
I would prefer `if (isset($objStopwatch))` in order to have less additional lines of code.

Reply:
Fixed in a22fa63fee3d8c139a3c04669804833fddff37c2.